Makefile

```diff
@@ -89,7 +89,7 @@ test-flake8:
 
 .PHONY: test-ruff
 test-ruff:
-	ruff rockcraft tests
+	ruff check rockcraft tests
```

Comment:
since we're dropping `ruff` from the requirements, can we check whether ruff is installed here? With a message on how to install it